### ISSUE TITLE
can't open joplin-cli on Ubuntu 20.04.1: "... /libm.so.6: version `GLIBC_2.29' not found"

### ISSUE THREAD

**Issue by manihamidi** [laurent22/joplin#4389]
I'm trying to use the joplin cli but can't open it despite a successful install on Ubuntu 20.04.1. I get the following:

```
me@home:-$ joplin
internal/modules/cjs/loader.js:1122
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: /snap/core/current/lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /home/mani/.joplin-bin/lib/node_modules/joplin/node_modules/sqlite3/lib/binding/napi-v6-linux-x64/node_sqlite3.node)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1122:18)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/home/mani/.joplin-bin/lib/node_modules/joplin/node_modules/sqlite3/lib/sqlite3-binding.js:4:15)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
```

... and I appear to have the latest version (2.31) installed: 

```
me@home:-$ ldd --version
ldd (Ubuntu GLIBC 2.31-0ubuntu9.1) 2.31
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```
## Environment

Joplin version: GUI is 1.6.7 but i can't figure out what the cli version is since I can't open it. 
Ubuntu 20.04.1

## Steps to reproduce

I just followed the install and open instructions on your page: https://joplinapp.org/terminal/

## Describe what you expected to happen

open the joplin cli app. 

## Logfile

I tried opening the cli using `joplin --debug --log-level debug` but i get the same error and above and therefore get no `~/.config/joplin` folder build to paste the logs here.

**Comment by manihamidi** [comment]
> What node version is it?

v14.15.4

**Comment by manihamidi** [comment]
seems to be an issue with node's snap container? https://github.com/JoshuaWise/better-sqlite3/issues/403

**Comment by manihamidi** [comment]
I just tried this again on another ubuntu machine and the install and launch worked. 

Unfortunately, I can't know for sure what made things work on this new computer, but here are some differences that I do notice between the two: 

* I had installed node using nvm on this machine, and used node version 15.10.0 and npm version 7.5.3. 
* My Ubuntu version is 20.04.2 which is one minor update above what i was using before. 
* Joplin GUI i have installed is at 1.7.11, but this whole thread had to do with the installation of the CLI which i think is independent. 

Note that the GLIBC I had installed had not changed: it is still `ldd (Ubuntu GLIBC 2.31-0ubuntu9.1) 2.31`

I think the first one may have been important, because I had trouble installing things with a system-installation of node until I switched to using nvm: https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally
